src/status_im2/common/nickname_drawer/view.cljs

```diff
@@ -0,0 +1,79 @@
+(ns status-im2.common.nickname-drawer.view
```

Comment:
somewhat of a nit, but I don't see what's "common" about this. It's not a very reusable component -> perhaps it would be better to put this into the namespace `status-im2.contexts.contacts.drawers.nickname-drawer.view` as it's mostly related to contact type pieces